src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -153,7 +154,7 @@ public class StrykerOptions
         /// Context: When using the since feature, all tests are run again if files in the test project change (as these could impact the test results)
         /// When the file is present in this option the tests should not run again as the file does not impact test results.
         /// </summary>
-        public IEnumerable<FilePattern> DiffIgnoreChanges { get; init; } = Enumerable.Empty<FilePattern>();
+        public IEnumerable<ExcludableString> DiffIgnoreChanges { get; init; } = Enumerable.Empty<ExcludableString>();
```

Comment:
This is where we really only need the excludability concept, spans are not taken into account by the diff provider.